### ISSUE TITLE
overflow in ai attack calculation

### ISSUE THREAD

**Issue by mray271** [wesnoth/wesnoth#2278]
Getting a crash in turn 6 while playing Trinity 1 during Storming Weldyn with 24 Active Troops 5 Reserve Troops on a Linux 4.13.16-300.fc27.x86_64  GNU/Linux

Error is:
20171205 20:59:51 warning ai/general: overflow in ai attack calculation
wesnoth: src/attack_prediction.cpp:1764: double {anonymous}::calculate_probability_of_debuff(double, bool, double, double, bool, double): Assertion `prob_stay_alive >= 0.0 && prob_stay_alive <= 1.0' failed.

wesnoth --version
Battle for Wesnoth v1.13.10
Started on Tue Dec  5 21:10:12 2017

Battle for Wesnoth 1.13.10

Library versions:
Boost:             1.64
OpenSSL/libcrypto: 1.1.0f  (runtime 1.1.0g)
Cairo:             1.15.8  (runtime 1.15.8)
Pango:             1.40.12 (runtime 1.40.13)
SDL:               2.0.6   (runtime 2.0.7)
SDL_image:         2.0.1   (runtime 2.0.2)
SDL_mixer:         2.0.1   (runtime 2.0.2)
SDL_ttf:           2.0.14  (runtime 2.0.14)
libpng:            1.6.31  (runtime 1.6.31)

Optional features:
Experimental OpenMP support:    no
PNG screenshots:                yes
Lua console completion:         no
Legacy bidirectional rendering: yes
D-Bus notifications back end:   no

**Comment by Vultraz** [comment]
Ah, wait, are you using the official 1.13.10 release? If so, this is probably already fixed in master.

**Comment by Pentarctagon** [comment]
If it were build from master, it would display something like this instead:
```
./wesnoth --version 
Battle for Wesnoth v1.13.10+dev (aa94db4-Modified)
```

**Comment by Vultraz** [comment]
Closing this, then. Was fixed in #2251.

**Comment by CelticMinstrel** [comment]
You mean it's a _duplicate_ of #2251, since that's an issue not a PR>
